### PR TITLE
Upgrade nginx to 1.23.3

### PR DESCRIPTION
**Pull request by zacblazic**
See [changelog](https://nginx.org/en/CHANGES) for what's changed since v1.19.5.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.23.3
+
+* Upgrade to nginx 1.23.3.
```
Comment (itskingori) [inline CHANGELOG.md:5]: Could you include some of the other changes that may be material?

File: config/nginx.conf [hunk 2]
```diff
@@ -1,6 +1,7 @@
+user  nginx;
```
Comment (itskingori) [inline config/nginx.conf:1]: What is this for?

File: config/nginx.conf [hunk 3]
```diff
@@ -1,6 +1,7 @@
...
 worker_processes  auto;
 
-error_log  /var/log/nginx/error.log warn;
+error_log  /var/log/nginx/error.log notice;
```
Comment (itskingori) [inline config/nginx.conf:4]: What your thinking here?

**Review by itskingori — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (zacblazic) [reply CHANGELOG.md:5]: All changes are related to upgrading.

**Comment by itskingori** [comment]
Oops, dismissed. I thought it was set to comment. 🙈

File: CHANGELOG.md (hunk 1 above)
Comment (itskingori) [reply CHANGELOG.md:5]: Ah, ok. 👍🏽

File: config/nginx.conf (hunk 3 above)
Comment (zacblazic) [reply config/nginx.conf:4]: This file is an exact copy of the upstream image's built-in config, as per commit message. Usually you'd provide your own `nginx.conf`.

**Review by itskingori — APPROVED** [review]

File: config/nginx.conf (hunk 3 above)
Comment (itskingori) [reply config/nginx.conf:4]: Noted. Good point.

File: config/nginx.conf (hunk 2 above)
Comment (itskingori) [reply config/nginx.conf:1]: Closing, see https://github.com/Intellection/docker-nginx/pull/9#discussion_r1101154086.

Comment (zacblazic) [reply config/nginx.conf:1]: Sets the user for worker processes. See [`user` directive](https://nginx.org/en/docs/ngx_core_module.html#user). Also a copy of upstream's config.

**Pull request merged**